system/jlib/jstats.h

```diff
@@ -928,6 +928,19 @@ class jlib_decl NullStatisticTarget : implements IStatisticTarget
     }
 };
 
+class jlib_decl RuntimeStatisticTarget : implements IStatisticTarget
+{
+public:
+    RuntimeStatisticTarget(CRuntimeStatisticCollection & _target) : target(_target) {}
+
+    virtual void addStatistic(StatisticScopeType scopeType, const char * scope, StatisticKind kind, char * description, unsigned __int64 value, unsigned __int64 count, unsigned __int64 maxValue, StatsMergeAction mergeAction)
```

Comment:
perhaps this is for future overloading, but most args unused ..

Reply:
I'll add override to clarify